src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.tsx

```diff
@@ -147,6 +149,8 @@ function AttachmentPickerWithMenuItems({
     const [allBetas] = useOnyx(ONYXKEYS.BETAS, {canBeMissing: true});
     const isASAPSubmitBetaEnabled = Permissions.isBetaEnabled(CONST.BETAS.ASAP_SUBMIT, allBetas);
     const hasViolations = hasViolationsReportUtils(undefined, transactionViolations);
+    const [allReports] = useOnyx(ONYXKEYS.COLLECTION.REPORT, {canBeMissing: true});
```

Comment:
Same question as above. Are we going to create a performance problem by subscribing this component to all reports?